include/openssl/stack.h

```diff
@@ -492,12 +497,26 @@ BSSL_NAMESPACE_END
                          (OPENSSL_sk_delete_if_func)func, data);               \
   }                                                                            \
                                                                                \
-  OPENSSL_INLINE int sk_##name##_find(const STACK_OF(name) *sk,                \
+  /* use 3-arg sk_*_find_awslc when size_t-sized |out_index| needed */         \
+  OPENSSL_INLINE int sk_##name##_find_awslc(const STACK_OF(name) *sk,          \
                                       size_t *out_index, constptrtype p) {     \
     return OPENSSL_sk_find((const OPENSSL_STACK *)sk, out_index,               \
                            (const void *)p, sk_##name##_call_cmp_func);        \
   }                                                                            \
                                                                                \
+  /* use 2-arg sk_*_find for OpenSSL compatibility */                          \
+  OPENSSL_INLINE int sk_##name##_find(const STACK_OF(name) *sk,                \
+                                      constptrtype p) {                        \
+    size_t out_index = 0;                                                      \
+    int ok = OPENSSL_sk_find((const OPENSSL_STACK *)sk, &out_index,            \
+                             (const void *)p, sk_##name##_call_cmp_func);      \
```

Comment:
Nit: Can this call `sk_##name##_find_awslc`? No strong preference though

Reply:
that would work, but IMO we should put any checks, etc. we want shared by both `sk_*_find` and `sk_*_find_awslc` in the concrete function `OPENSSL_sk_find` rather than one of the wrapping macros.